src/components/attendee/ProfileOverlay/ProfileOverlay.module.scss

```diff
@@ -41,3 +41,7 @@
     line-height: 1.4;
   }
 }
+
+.logOut {
+  margin-bottom: space.empty(3);
```

Comment:
`space.button(1)` is effectively `space.empty(3.5)` for uniformity reasons

Reply:
[84cb042](https://github.com/sparkletown/sparkle/pull/3030/commits/84cb042903c42e21a52cb03d0e89e0b6aed4cdcb)